mrp_subcontracting_inhibit/models/product_supplierinfo.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductSupplierinfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    subcontracting_inhibit = fields.Boolean(string="Inhibit subcontracting")
```

Comment:
Give better context:

```suggestion
    subcontracting_inhibit = fields.Boolean(string="Subcontracting inhibited")
```